### ISSUE TITLE
DeviantArt downloader malfunctions [Note: Artstation report has been moved to a new issue]

### ISSUE THREAD

**Issue by UnforeseenOcean** [RipMeApp/ripme#1398]
* Ripme version: 1.7.86
* Java version: 1.8.0_211
* Operating system: 6.1.7601
* Exact URL you were trying to rip when the problem occurred: Multiple, listed below, may contain NSFW images
```
https://www.deviantart.com/ralphhorsley/gallery/?catpath=/
https://www.deviantart.com/orcbruto/gallery/?catpath=/
https://www.deviantart.com/jamesryman/gallery/?catpath=/
https://www.deviantart.com/rntentn/gallery/?catpath=/
https://www.deviantart.com/rogierb/gallery/?catpath=/
https://www.artstation.com/maxbanshees
```
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

The images get downloaded.

## Actual Behavior
ArtStation links get skipped immediately, and DeviantArt downloader spews errors such as "No image found in (link)"

**Comment by rautamiekka** [comment]
DA's malfunctions are a known thing already. The devs are waiting for DA to finish up the system. Frankly, I doubt they'll get it done anytime soon, which is increasingly a massive problem.

**Comment by UnforeseenOcean** [comment]
Yeah. Now I'm wondering if a custom login function, which was requested a
while ago, could be implemented. That might cut down on the rate limit
problem (if that was the issue)

**Comment by UnforeseenOcean** [comment]
I'm wondering why the ArtStation links are skipped without leaving any log behind.

**Comment by SavageCourier** [comment]
I have a similar problem I try to download from the DeviantArt gallery of my favorite artist but all I get is null

**Comment by UnforeseenOcean** [comment]
I still get the null error on DeviantArt downloader and ArtStation links are just straight-up ignored. I gave it 50 or so links and it ignored all of them.

**Comment by Owyn** [comment]
```
Downloading https://www.deviantart.com/kittew/gallery/all
Rip complete, saved to C:\DownLoads\rips\deviantart_kittew_gallery_unknown
null
```
same, was looking for a DA downloader, but no luck so far ;_;
